crates/next-api/src/nft_json.rs

```diff
@@ -95,6 +100,81 @@ fn get_output_specifier(
     bail!("NftJsonAsset: cannot handle filepath {}", path_ref);
 }
 
+/// Apply outputFileTracingIncludes patterns to find additional files
+async fn apply_includes(
+    project_root_path: Vc<FileSystemPath>,
+    includes: &BTreeSet<RcStr>,
+    ident_folder: &FileSystemPath,
+) -> Result<BTreeSet<RcStr>> {
+    let mut result = BTreeSet::new();
+
+    for include_glob in includes {
+        // Read files matching the glob pattern from the project root
+        // let project_root = project_fs.root();
+        let project_root_path_ref = project_root_path.await?;
+        let glob_result = project_root_path
+            .read_glob(Glob::new(include_glob.clone()), true)
+            .await?;
+
+        // Process the glob results recursively
+        Box::pin(collect_glob_results(
+            &glob_result,
+            "",
+            &mut result,
+            ident_folder,
+            &project_root_path_ref,
+        ))
+        .await?;
```

Comment:
compose into a single Glob via the `Glob::alternatives` API so you can do a single read